op-node/rollup/derive/attributes.go

```diff
@@ -136,6 +136,17 @@ func (ba *FetchingAttributesBuilder) PreparePayloadAttributes(ctx context.Contex
 			return nil, NewCriticalError(fmt.Errorf("failed to create depositsCompleteTx: %w", err))
 		}
 		afterForceIncludeTxs = append(afterForceIncludeTxs, depositsCompleteTx)
+
+		newDependencies := ba.rollupCfg.IsDependencySetUpdate(l2Parent.Time, nextL2Time)
```

Comment:
Apologies here, I still think we should return 2 args

```go
deps, isUpdate := ba.rollupCfg.IsDependencySetUpdate(...); if isUpdate { ...
```

We can keep the approach of 1 tx per update